UKHomeOffice/nmsw-ui
# NMSW-460 Populate general declaration section of CYA from FAL 1 data

# Ticket

NMSW-460

----

## AC

Given I have uploaded a FAL 1 form
When I view the Check Your Answers page
Then the details from my FAL 1 form are displayed on that page

----

## To test

- create a voyage report
- upload a General Declaration file
- on the task details page click on the check your answer link (it is enabled for testing purposes)
- > you should be taken to the cya page
- > you should see the general declaration section values match the values in the file you uploaded

----

## Notes
